### DISCUSSION TITLE
regional pricing

### DISCUSSION THREAD

**Discussion by bingoxo** [ente-io/ente#2598] [Enhancements]
foss  private secure alternative to google photos is great but it's hard to justify it when there is huge price difference since Google offers regional pricing , i know you can't be cheaper than Google (fingers crossed) , but i hope you can bring the prices down further (i know you did this week but it's still expensive)

**Comment by brog-io** [comment]
Regional pricing isn't possible, the costs are fixed. The reason why Google can be so cheap is because they make their money on your data. They probably make a loss if they don't.

**Reply by bingoxo** [reply]
Google can afford to be cheap and i don't expect ente (or anyone else) to be as cheap , but at same time there are other foss projects who don't sell your data and still affordable like filen.io (e2ee cloud storage) , hopefully ente can find cheaper storage providers
